scraping/reddit/utils.py

```diff
@@ -95,23 +94,22 @@ def validate_reddit_content(
             content_size_bytes_validated=entity_to_validate.content_size_bytes,
         )
 
-    # Check Reddit created_at
+    # Timestamps on the contents within the entities must be obfuscated to the minute.
```

Comment:
Also check the actual created_at matches the entity.datetime like we do for twitter now?